README.md

```diff
@@ -7,6 +7,32 @@ Google APIs and help you to utilize them more efficiently. You can also
 use these definitions with open source tools to generate client
 libraries, documentation, and other artifacts.
 
+## Building
+### Bazel
+
+The recommended way to build the API client libraries is through
+[Bazel](https://bazel.build/) >= 0.23.0.
+
+To build all libraries:
+
+```
+bazel build //...
+```
+
+To test all libraries:
+
+```
+bazel test //...
+```
+
+To build one library in one language:
```

Comment:
This is somewhat inaccurate. That target will crate a bazel-independent package (almost identical to what artman generates, the one with gradle scripts in it for java).  To build single lisbrary it would be something like:
```
bazel build //google/example/library/v1/...
```
That would build all target in the library (including the `google-cloud-library-v1-java` target).

I belive it is worth having both targets as separate items in the readme (i.e. so users know how to build everything for a single library and also know how to get artman-like pakcage for the single library explicitly).

Reply:
Done

Reply:
@vam-google Is package more accurate?